tests/functional/test_route53.py

```diff
@@ -28,6 +28,23 @@ def setUp(self):
             'MaxItems': '1'
         }
         self.operation_name = 'list_hosted_zones'
+        self.final_params = {}
+
+    def _store_final_params(self, params, **kwargs):
+        self.final_params = params
+
+    def test_route53_id_conversion(self):
+        # This conversion is handled in the before-parameter-build event, but
```

Comment:
Thoughts on using the HTTP stubber instead?

Reply:
I thought about this but since this ends up as a query arg I'd have to parse the request URL and pull out the right arg to assert on. It felt more wrong than just hooking into an event that would have the correctly converted parameter.

Reply:
I thought about this but since this ends up as a query arg I'd have to parse the request URL and pull out the right arg to assert on. It felt more wrong than just hooking into an event that would have the correctly converted parameter.

Reply:
Fair enough.